### PR TITLE
Us146b 2 implement auth

### PR DESCRIPTION
**Pull request by Kirosoft**

### CODE REVIEW

**Comment by danielmarker** [comment]
Good piece of work here :+1:

File: run.py [hunk 1]
```diff
@@ -8,4 +8,4 @@
 
 LOGGER.info("Starting the server")
 
-app.run(debug=True, host="0.0.0.0", port=int(os.getenv("PORT", "5000")))
+app.run(debug=True, host="0.0.0.0", port=int(os.getenv("PORT", "5005")))
```
Comment (tsangsimon) [inline run.py:11]: Why are we changing the port?

**Comment by tsangsimon** [comment]
Looks good but needs more LOGGER.info statements when processing the headers. I would also put a LOGGER.error block in when the it is unable to process the headers (401 unauthorised)  and at the 400 response

File: run.py (hunk 1 above)
Comment (Kirosoft) [inline run.py:11]: The defaults for all our apps are set at 5000, so they conflict. Changing this means we can start the apps in the dev vm without needed override switches.

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**